kotlin-audio-sample/src/main/java/com/doublesymmetry/kotlin_audio_sample/utils/FakeData.kt

```diff
@@ -1,18 +1,18 @@
-package com.doublesymmetry.kotlin_audio_sample
+package com.doublesymmetry.kotlin_audio_sample.utils
 
 import com.doublesymmetry.kotlinaudio.models.DefaultAudioItem
-import com.doublesymmetry.kotlinaudio.models.SourceType
+import com.doublesymmetry.kotlinaudio.models.MediaType
 
 
 val firstItem = DefaultAudioItem(
-    "https://www.learningcontainer.com/wp-content/uploads/2020/02/Kalimba.mp3", SourceType.FILE,
```

Comment:
The `MediaType` is not used anymore, as the player works automatically with both local files and URLs. I'll make a low priority ticket for this.

Reply:
`MediaType` is for HLS/DASH, etc support :) not for local/remote files